templates/emails/account_new.html

```diff
@@ -0,0 +1,174 @@
+<!doctype html>
```

Comment:
You should make the email templates inherit from an emails/base.html that has all the head/styling in it, to avoid this repetition.

Reply:
Can do! Im working on this template now for the moderation emails so i can strip default identity into a separate PR and work on the inheritance for all html templates